app/views/posts/index.html.erb

```diff
@@ -15,6 +15,8 @@
   <tbody>
     <% @posts.each do |post| %>
       <tr>
+        <% @post = post %>
+        <td><%= render 'vote_form'%></td>   
```

Comment:
Ugh, tables are so 2000. But that's fine for now. We can change it when we style it.